samples/tetris/build.bat

```diff
@@ -9,12 +9,11 @@ set MINGW=\msys64\mingw64
 
 set "CFLAGS=-I%MINGW%\include\SDL2"
 rem Add -Wl,--subsystem,windows for making GUI subsystem application.
-set "LFLAGS=%DIR%\Tetris.res -L%MINGW%\lib -lSDL2"
+set "LFLAGS=%DIR%\Tetris.res -L%MINGW%\lib -Wl,-Bstatic -lstdc++ -static -lSDL2 -limm32 -lole32 -loleaut32 -lversion -lwinmm -mwindows"
```

Comment:
Does Gradle build produce dynamically linked binary?

Reply:
Gradle part was merged as part of LLVM6 - so now it produces static binary, but still copy unused SDL2.dll